enumeratum/src/main/scala/com/monovore/decline/enumeratum/ValueEnumArgument.scala

```diff
@@ -18,7 +18,7 @@ private[enumeratum] final class ValueEnumArgument[A, Entry <: ValueEnumEntry[A]]
       case Validated.Valid(value) =>
         enum.withValueOpt(value) match {
           case Some(r) => Validated.validNel(r)
-          case None => Validated.invalidNel(s"Invalid value: $string")
+          case None => Validated.invalidNel(invalidChoice(string, enum.values.map(_.value.toString)))
```

Comment:
Had to check that `toString` is an inverse of parsing for all the basic value types, but that seems to be true!

Reply:
Thank you! I've also checked that. It maybe is better to require implicit Show here but I believe it would break binary compatibility

Reply:
Checked that it does not - ValueEnumArgument is private, and all the cases are listed in the package object.